premake5.lua

```diff
@@ -243,6 +245,14 @@
 			buildoptions { "-arch arm64", "-arch x86_64" }
 			linkoptions { "-arch arm64", "-arch x86_64" }
 
+		filter { "system:macosx", "options:arch=ppc or arch=ppc" }
+			buildoptions { "-arch ppc" }
+			linkoptions { "-arch ppc" }
+
+		filter { "system:macosx", "options:arch=ppc64 or arch=ppc64" }
```

Comment:
```suggestion
		filter { "system:macosx", "options:arch=ppc64" }
```

Reply:
@Jarod42 Are these equivalent? I just copied the logic here from x86/arm cases, where both are used.

Reply:
Ah, okay, I misunderstood what it does. Will fix this now.